### DISCUSSION TITLE
How can I update EasyChat?

### DISCUSSION THREAD

**Discussion by Aholicknight** [Earu/EasyChat#115] [Q&A] [answered]
I know this is a dumb question, but I downloaded this repo with `git clone https://github.com/Earu/EasyChat` and there has been an update to EasyChat. How would I update it? Do I have to delete EasyChat and clone everytime there is an update?

**Comment by Codingale** [comment, marked as answer]
run `git pull` in the directory

**Reply by Earu** [reply]
What the gentleman said!

**Reply by Aholicknight** [reply]
Thanks!
